wagtail/admin/static_src/wagtailadmin/css/normalize.css

```diff
@@ -450,7 +450,7 @@ input[type="submit"] {
 input[type='checkbox'],
 input[type='radio'] {
   box-sizing: border-box; /* 1 */
-  padding: 0; /* 2 */
+  padding: 0 5px; /* 2 */
```

Comment:
This padding makes hard assumption about the zoom and font size. 

Maybe we should explore a more robust solution. 
em/rem units could help. 

Also. It might be worth fixing the checkbox heights to a similar approach.  

Please also check other styles for checkbox styling. This is normalize so there may be widget specific styles to change instead of.

Reply:
That makes sense. Okay, so I'll work on relative padding/styling and some other minor changes 👍